src/records/Course.js

```diff
@@ -0,0 +1,26 @@
+import {Record} from 'immutable';
+
+export default class Course extends Record({
+  id: null,
+  name: null,
+  section: null,
+  descriptionHeading: null,
+  description: null,
+  room: null,
+  ownerId: null,
+  creationTime: null,
+  updateTime: null,
+  enrollmentCode: null,
+  courseState: null,
+  alternateLink: null,
+  teacherGroupEmail: null,
+  courseGroupEmail: null,
+  guardiansEnabled: null,
+  calendarId: null,
+}, 'Course') {
+  static fromJS(js) {
+    return new this(
+      Object.assign({}, js),
```

Comment:
Do we need the `Object.assign`? `Immutable.Record`’s constructor certainly shouldn’t mess with the argument it’s passed…